packages/block-editor/src/components/segmented-text-control/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { BaseControl, Button } from '@wordpress/components';
+
+/**
+ * @typedef {Object} Option
+ * @property {string} label The label of the control.
+ * @property {string} value The value of the control.
+ * @property {string} icon  The icon of the control.
+ */
+
+/**
+ * Control to facilitate selecting a text style from a set of options.
+ *
+ * @param {Object}   props           Component props.
+ * @param {string}   props.label     A label for the control.
+ * @param {string}   props.value     Currently selected value.
+ * @param {Function} props.onChange  Callback to handle onChange.
+ * @param {Option[]} props.controls  Array of controls to display.
+ * @param {string}   props.className Additional class name to apply.
+ *
+ * @return {Element} Element to render.
+ */
+export default function SegmentedTextControl( {
+	label,
+	value,
+	controls,
```

Comment:
It seems like we've sort of gone only halfway to renaming these controls to options.

As I see it, the overall collection of options is the "control", they all set the one style. Each button reflects a different possible value or "option".